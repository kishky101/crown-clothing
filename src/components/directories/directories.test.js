import renderer from 'react-test-renderer';
import Directories from './directories.component';
import { BrowserRouter as Router } from 'react-router-dom';

test('creating directorie snapshot', () => {
    const directories = renderer.create(
        <Router>
            <Directories />
        </Router>
    ).toJSON;
    expect(directories).toMatchSnapshot();
})