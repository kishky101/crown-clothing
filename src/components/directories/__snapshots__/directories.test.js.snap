// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creating directorie snapshot 1`] = `[Function]`;
